### PR TITLE
fix(guide): Replace invalid language postfix for code blocks in Guide (across all languages)

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #35960

I was able to get rid of all but two of the warnings about `cs` and `c++`.  I used the following regular expressions (in VS Code) and replaced several `cs` with `csharp` and replaced sevral `c++` with `cpp`.
**cs**
````
```cs\r?\n
````
**c++**
````
```c\+\+
````

### CODE REVIEW

**Comment by RandellDawson** [comment]
OK, this PR is ready for review.

File: guide/portuguese/csharp/hello-world/index.md [hunk 1]
```diff
@@ -28,7 +29,7 @@ using System;
 ```
 
 ## Saída:
-
+```shell
 ```sh
```
Comment (huyenltnguyen) [inline guide/portuguese/csharp/hello-world/index.md]: I'm wondering if this line should be removed.

File: guide/spanish/csharp/hello-world/index.md [hunk 2]
```diff
@@ -28,7 +29,7 @@ using System;
 ```
 
 ## Salida:
-
+```shell
 ```sh
```
Comment (huyenltnguyen) [inline guide/spanish/csharp/hello-world/index.md]: Same as above.

File: guide/spanish/rust/installing-rust/index.md [hunk 3]
```diff
@@ -12,7 +13,7 @@ Visite el [sitio web de rustup](https://rustup.rs) y descargue el `rustup-init.e
 ## Instalación de Rust en otros sistemas operativos (Mac OS X, Linux, BSD, Unix)
 
 Abre tu terminal y escribe este comando:
-
+```shell
 ```sh
```
Comment (huyenltnguyen) [inline guide/spanish/rust/installing-rust/index.md]: Same as above.

File: guide/spanish/rust/installing-rust/index.md [hunk 4]
```diff
@@ -22,7 +23,7 @@ Esto buscará el instalador de `rustup` y, a su vez, `rustup` todo lo que necesi
 # Verificando instalación
 
 La instalación de `rustup` instalará todas las cosas relevantes para la oxidación, pero lo más importante es que se instale el compilador y el administrador de paquetes. Para verificar que todo está instalado, ejecute este comando:
-
+```shell
 ```sh
```
Comment (huyenltnguyen) [inline guide/spanish/rust/installing-rust/index.md]: Same as above.

File: guide/portuguese/csharp/hello-world/index.md (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply guide/portuguese/csharp/hello-world/index.md]: Yes, I believe ```sh should be removed.

Comment (RandellDawson) [reply guide/portuguese/csharp/hello-world/index.md]: I have fixed this in my latest commit.

File: guide/spanish/rust/installing-rust/index.md (hunk 3 above)
Comment (RandellDawson) [reply guide/spanish/rust/installing-rust/index.md]: I have fixed this in my latest commit.

File: guide/spanish/csharp/hello-world/index.md (hunk 2 above)
Comment (RandellDawson) [reply guide/spanish/csharp/hello-world/index.md]: I have fixed this in my latest commit.

File: guide/spanish/rust/installing-rust/index.md (hunk 4 above)
Comment (RandellDawson) [reply guide/spanish/rust/installing-rust/index.md]: I have fixed this in my latest commit.

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Comment by scissorsneedfoodtoo** [comment]
Awesome! This all looks great. Thank you for taking the time to fix all of these articles, @RandellDawson.

**Pull request merged**